### PR TITLE
Add test for getting Control Center

### PR DESCRIPTION
**Pull request by imurchie**

### CODE REVIEW

File: test/functional/basic/gesture-e2e-specs.js [hunk 1]
```diff
@@ -223,6 +223,29 @@ describe('XCUITestDriver - gestures', function () {
         await doPinch();
       });
     });
+    describe('special actions', function () {
+      it('should open the control center by swiping up at the bottom', async function () {
+        await driver.elementByAccessibilityId('ControlCenterView')
+          .should.eventually.be.rejectedWith(/An element could not be located/);
+
+        const window = await driver.elementByClassName("XCUIElementTypeWindow");
```
Comment (mykola-mokhnach) [inline test/functional/basic/gesture-e2e-specs.js]: single quotes

File: test/functional/basic/gesture-e2e-specs.js [hunk 2]
```diff
@@ -223,6 +223,29 @@ describe('XCUITestDriver - gestures', function () {
...
+        const size = await window.getSize();
```
Comment (mykola-mokhnach) [inline test/functional/basic/gesture-e2e-specs.js]: `const {width, height}`

File: test/functional/basic/gesture-e2e-specs.js [hunk 3]
```diff
@@ -223,6 +223,29 @@ describe('XCUITestDriver - gestures', function () {
...
+
+        let action = new wd.TouchAction(driver);
+        action.press({
+          x: size.width / 2,
+          y: size.height - 5,
+        }).wait(500)
```
Comment (mykola-mokhnach) [inline test/functional/basic/gesture-e2e-specs.js]: is 500 ms enough?

File: test/functional/basic/gesture-e2e-specs.js [hunk 4]
```diff
@@ -223,6 +223,29 @@ describe('XCUITestDriver - gestures', function () {
...
+        .moveTo({
+          x: size.width / 2,
+          y: size.height / 2,
+        });
+        await action.perform();
+
+        // Control Center ought to be visible now
+        await driver.elementByAccessibilityId('ControlCenterView');
```
Comment (mykola-mokhnach) [inline test/functional/basic/gesture-e2e-specs.js]: should.be.fulfilled?

File: test/functional/basic/gesture-e2e-specs.js (hunk 3 above)
Comment (imurchie) [reply test/functional/basic/gesture-e2e-specs.js]: Seems to be locally and on Travis.

File: test/functional/basic/gesture-e2e-specs.js (hunk 4 above)
Comment (imurchie) [reply test/functional/basic/gesture-e2e-specs.js]: There's no particular point. If it throws an error the test will fail for it. If it is never resolved the test will time out.

File: test/functional/basic/gesture-e2e-specs.js [hunk 5]
```diff
@@ -223,6 +223,29 @@ describe('XCUITestDriver - gestures', function () {
         await doPinch();
       });
     });
+    describe('special actions', function () {
+      it('should open the control center by swiping up at the bottom', async function () {
+        await driver.elementByAccessibilityId('ControlCenterView')
```
Comment (jlipps) [inline test/functional/basic/gesture-e2e-specs.js]: is `ControlCenterView` something that xcuitest actually returns as the element type, or is it a special thing we made?

File: test/functional/basic/gesture-e2e-specs.js [hunk 6]
```diff
@@ -223,6 +223,29 @@ describe('XCUITestDriver - gestures', function () {
...
+        const {width, height} = await window.getSize();
+
+        let action = new wd.TouchAction(driver);
+        action.press({
+          x: width / 2,
+          y: height - 5,
+        }).wait(500)
+        .moveTo({
+          x: width / 2,
+          y: height / 2,
```
Comment (jlipps) [inline test/functional/basic/gesture-e2e-specs.js]: why not move all the way to the bottom?

File: test/functional/basic/gesture-e2e-specs.js (hunk 5 above)
Comment (imurchie) [reply test/functional/basic/gesture-e2e-specs.js]: In my experiments, that is what XCUITest returns for the thingo down there.

File: test/functional/basic/gesture-e2e-specs.js (hunk 6 above)
Comment (imurchie) [reply test/functional/basic/gesture-e2e-specs.js]: It's moving up, and there is no need to go to the top.

**Pull request merged**